scripts/get-tokens-from-invision-dsm.js

```diff
@@ -20,6 +21,11 @@ const fetchTokens = async () => {
 // Invision’s API returns colors under this path
 const getColorTokens = (object) => object.list.colors[0].colors;
 
+// local metadata
```

Comment:
Can you mention that this is a temporary workaround until DSM is resurfacing the `description` in its API?